pkg/quay/quay.go

```diff
@@ -34,19 +38,27 @@ func New(c Config) (*Quay, error) {
 	httpClient := &http.Client{}
 
 	if c.Namespace == "" {
-		return nil, microerror.Maskf(invalidConfigError, "Namespace must not be empty")
+		return nil, microerror.Maskf(invalidConfigError, "%T.Namespace must not be empty", c)
+	}
+
+	quayToken := os.Getenv(env.QuayApiToken)
```

Comment:
Didn't want to pass it as flag as that's kind of super-specific to quay and there is no way to retrieve this token with login/password

Reply:
Why not `--quay-api-token` flag then?

Reply:
I'm fine with the PR in general but I still want this flag with defaulting to env var value for discoverability reasons.